Assets/MixedRealityToolkit/_Core/Utilities/MathUtilities.cs

```diff
@@ -439,5 +439,19 @@ public static float RadiansToDegrees(float radians)
         {
             return (radians * Mathf.Rad2Deg);
         }
+
+        /// <summary>
+        /// Calculates the angle (at pointA) between two, two-dimensional points.
+        /// </summary>
+        /// <param name="pointA">The first point.</param>
+        /// <param name="pointB">The second point.</param>
+        /// <returns>
+        /// The angle between the two points.
+        /// </returns>
+        public static float GetAngleBetween(Vector2 pointA, Vector2 pointB)
```

Comment:
Is this different from Vector2.Angle? https://docs.unity3d.com/ScriptReference/Vector2.Angle.html If so, include the difference in the summary text, since it's not immediately apparent to me what the difference is.